src/gpu/intel/jit/gemm/generator/pieces/matrix_multiply.cxx

```diff
@@ -255,17 +252,12 @@ void BLASKernelGenerator<hw>::outerProductFMA(int h, int ha, int hb, int opCount
                     RegData bcastSrc = bcastSrcSub;
 
                     if (state.broadcast) {
```

Comment:
You can remove `state.broadcast` and everything related to it entirely. Besides Gen9, this was an old path to support mixed f32 * bf16 GEMMs but is not used any more.

Reply:
Done.